### DISCUSSION TITLE
Pix editor improvements

### DISCUSSION THREAD

**Discussion by aldenwalker** [linuxmint/discussions#343] [Ideas]
The new layout for the Pix editor now has the behavior that if you click an action (e.g. crop), the sidebar changes to show the crop inputs, and it hides the other edit tools.  So if you click crop and then decide you want to rotate, you must back out of the crop menu.  It used to be that all edit actions were visible all the time, so previously you could just click rotate to go directly there.  Also, backing out of the crop action requires either moving the mouse all the way to the far upper left or clicking on the image and then pressing esc.

There is also not (I think) a keyboard shortcut for "accept", so you must use the mouse to go to the top of the screen and click the button instead of just pressing enter or something.

Is the button size determined by the ambient linux mint theme?  They are rather smaller than before, but perhaps that is outside of Pix's control.

Am I missing some shortcuts for doing these things or increasing the button size?

**Comment by Jopp-gh** [comment]
I like Pix and its compact toolbar design, but, all those menubar buttons are just confusing and feel awkward in use.

Just remove all those buttons (maybe keep the presentation button). 

So when you click the hamburger button, make Pix open its sidebar panel on the right hand side with all the related commands organized in tabs.
 
Since I use Darktable almost daily and for many years now, I highly recommend you to check out how Darktable solved the Ux layout:
![darktable-0-2196747673](https://github.com/user-attachments/assets/af70faff-71e4-49e1-aab1-16211d5e819b)


BTW, Darktable and Pix are complementary, imo. The best features of Darktable are the raw development modules, not the digital asset management aspect.

**Comment by cplittlfield** [comment]
I stopped using Pix after they took away the menu bar, and I got tired of having to stop what I was doing and hunt for the option I wanted.  A well-designed user interface makes it explicit what options are available and what they do.  Having to stop and guess what some unlabeled icon does feels like a bad  design to this user.

**Reply by Jopp-gh** [reply]
@cplittlfield 
Icons should be the quickest way to interact with apps, but ofc icons can't cover all shortcuts.  

The problem is, Pix opens multiple windows (eg. tags, comments, tools, hamburger menu) instead to put simply all menu bar commands in the right side panel, together with most used shortcuts (usually icon buttons).

Pix isn't bad but needs some better ux design so all users are able access Pix's  app commands.

Another thing I noticed is, red, green, blue flags have two shortcuts each, which is strange - a flag has to be a simple toggle - you don't do anything complicated with flags. But flags like stars are quicker than assigning tags, at least after an import to sort good from bad pictures
